
https://gitlab.gnome.org/GNOME/libxml2/-/commit/2a357ab99e6f5c91
in libxml2 fixed the exit status of a call to xml2-config with an invalid
argument. libxslt continues to call xml2-config with an invalid argument to
check if it's working...

diff -wpruN '--exclude=*.orig' a~/configure.in a/configure.in
--- a~/configure.in	1970-01-01 00:00:00
+++ a/configure.in	1970-01-01 00:00:00
@@ -582,7 +582,7 @@ dnl make sure xml2-config is executable,
 dnl test version and init our variables
 dnl
 
-if ${XML_CONFIG} --libs print > /dev/null 2>&1
+if ${XML_CONFIG} --libs > /dev/null 2>&1
 then
 	XMLVERS=`$XML_CONFIG --version`
 	if test VERSION_TO_NUMBER(echo $XMLVERS) -ge VERSION_TO_NUMBER(echo $LIBXML_REQUIRED_VERSION)
